defaults/main.yml

```diff
@@ -7,8 +7,8 @@ postgres_group: postgres
 postgres_locale: C
 postgres_encoding: UTF8
 postgres_home_directory: "/var/lib/postgresHome"
-postgres_initdb: initdb
-postgres_pg_ctl: /usr/bin/pg_ctl
+postgres_initdb: "/usr/lib/postgresql/9.5/bin/initdb"
```

Comment:
is your role specifically for version `9.5` of the package? Otherwise please change this.